content/docs/dvclive/get-started.md

```diff
@@ -1,134 +1,129 @@
 # Get Started
 
 DVCLive is a Python library for logging machine learning metrics and other
-metadata in simple file formats, which is fully compatible with DVC.
+metadata in simple file formats which are fully compatible with DVC.
 
-<admon type="tip">
+## Set up DVCLive
 
-If you use one of the supported [ML Frameworks](/doc/dvclive/ml-frameworks), you
-can jump directly to its corresponding page.
+First of all, you need to add DVCLive to your Python code in order to start
+logging metrics.
 
-</admon>
-
-## Workflow
-
-### Initialize DVCLive
+In this example we are using [Keras](/doc/dvclive/ml-frameworks/keras) for
+training our model and we can add DVCLive as follows:
```

Comment:
Note that Hugginfacae and PyTorch seem to be more popular now.

Reply:
I added tabs with keras, huggingface and pytorch lightning . There is a link right bellow to other frameworks